src/99-appendices/06-meg-file-formats.md

```diff
@@ -77,9 +100,11 @@ sub-control01/
             sub-control01_ses-001_task-rest_run-01_channels.tsv
 ```
 
-After applying the MaxFilter pre-processing tool, files should be renamed with
-the corresponding label (e.g., `proc-sss`) and placed into a `derivatives`
-subfolder.
+### Sharing FIFF data *after* Maxwell filtering
+
+After applying Maxwell filtering (e.g., by using the MaxFilter software),
+files should be renamed with the corresponding label (e.g., `proc-sss`)
+and placed in a `derivatives` subfolder.
```

Comment:
@robertoostenveld ok for you to enforce to put these files in derivatives?

Reply:
This is not a change --> it was like this since 2018